todo_app/flask_config.py

```diff
@@ -2,7 +2,8 @@
 
 
 class Config:
-    """Base configuration variables."""
-    SECRET_KEY = os.environ.get('SECRET_KEY')
-    if not SECRET_KEY:
-        raise ValueError("No SECRET_KEY set for Flask application. Did you follow the setup instructions?")
+    def __init__(self):
+        """Base configuration variables."""
+        self.SECRET_KEY = os.environ.get('SECRET_KEY')
+        if not self.SECRET_KEY:
+            raise ValueError("No SECRET_KEY set for Flask application. Did you follow the setup instructions?")
```

Comment:
Your e2e tests are currently failing in your pipeline because you aren't providing `-e SECRET_KEY=something` in your docker run command. The value doesn't matter at all, but it needs to be set.